Bundle/Tests/DependencyInjection/AmqpConfigurationTest.php

```diff
@@ -99,6 +99,8 @@ public function driverRestrictions($value, $valid)
                 $this->fail('Valid driver failed');
             }
 
+            $this->assertTrue(true);
```

Comment:
What's that?

Reply:
as I understand it, this test is successful, and it is necessary to miss it, which I do in the next commit.